paddle/framework/backward.cc

```diff
@@ -387,11 +396,33 @@ void AppendBackward(ProgramDescBind& program_desc,
     no_grad_var_names.insert(GradVarName(name));
   }
   const int root_block_idx = 0;
-  auto backward_op_descs =
-      MakeBlockBackward(program_desc, root_block_idx, no_grad_var_names);
-  auto& forw_op_descs = program_desc.Block(root_block_idx)->ops_;
+  auto backward_op_descs = MakeBlockBackward(program_desc, root_block_idx,
```

Comment:
target does not need to pass to `MakeBlockBackward`.

Reply:
removed